modules/digital-twins/digital-twins-instance/main.bicep

```diff
@@ -233,6 +228,14 @@ output location string = digitalTwinsInstance.location
 //   Definitions   //
 // =============== //
 
+type managedIdentitiesType = {
+  @description('Optional. Enables system assigned managed identity on the resource.')
+  systemAssigned: bool?
```

Comment:
As system assigned appears supported, let's add the corresponding principal ID output

Reply:
done